docs/configdoc.md

```diff
@@ -0,0 +1,117 @@
+# L3AFD Config Options Documentation
+
+See [l3afd.cfg](https://github.com/l3af-project/l3afd/blob/main/config/l3afd.cfg) for a full example configuration.
+
+
+```
+[DEFAULT]
+
+[l3afd]
+pid-file: ./l3afd.pid
+datacenter: dummy
+bpf-dir: /dev/shm
+bpf-log-dir:
+kernel-major-version: 4
+kernel-minor-version: 15
+shutdown-timeout: 1s
+http-client-timeout: 10s
+max-nf-restart-count: 3
+bpf-chaining-enabled: true
+swagger-api-enabled: false
+# PROD | DEV
+environment: PROD
+....
+```
+
+### Below is the detailed documentation for each field
+
+
+## [l3afd]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|pid-file| `"l3afd.pid"`  |It contains process id of eBPF Programs |
+|datacenter| `"dummy"` | Name of Datacenter|
+|bpf-dir| `"/dev/shm"` | Absolute Path of eBPF Programs (where they are extracted) |
+|bpf-log-dir|`""`      | Absolute Path,where we store logs eBPF Programs|
+|kernel-major-version|`"4"`|Major version of the kernel|
+|kernel-minor-version|`"15"`|Miner version of the kernel (Ex 4.15)|
+|shutdown-timeout|`"1s"`|It is the amount of time allowed to stop all network functions|
+|http-client-timeout|`"10s"`|It is the amount of time allowed to get response headers|
+|max-nf-restart-count|`"3"`|Maximum number of attempts to monitor eBPF Program|
+|bpf-chaining-enabled|`"true"`|Boolean to set bpf-chaining |
+|swagger-api-enabled|`"false"`|Boolean to check swagger api enable or not|
+|environment|`"PROD"`| Environment  variable |
+|BpfMapDefaultPath|`"/sys/fs/bpf"`|It is a base path for storing bpf maps|
+|TcMapsRelativePath|`"tc/globals"`|It is a relative path of tc maps with respect to BpfMapDefaultPath|
+## [kf-repo]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|url| `"http://localhost:8000/"`|Default Repository to Download eBPF Programs|
+
+## [web]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|metrics-addr|`"0.0.0.0:8898"`|Prometheus endpoint for pull/scrape the metrics|
+|kf-poll-interval|`"30s"`|Periodic Interval to scrape metrics using Prometheus|
+|n-metric-samples|`"20"`|Number of Metric Samples|
+
+## [admind]
+| FieldName     | Example       | Description     |
+| ------------- | ------------- | --------------- |
+|host|`""`|Hostname of admin|
+|username|`""`|Username of admin|
+|api-key|`""`|Api key of admin|
+|groud-id|`"0"`|Group id of admin|
```

Comment:
What is this used for?  Does it apply to Windows too and if so what format should group-id be in?